trapper/metrics/metric_handlers/question_answering_handler.py

```diff
@@ -9,6 +9,17 @@
 
 @MetricHandler.register("question-answering")
 class MetricHandlerForQuestionAnswering(MetricHandler):
+    """
+    `MetricHandlerForQuestionAnswering` provides converting the predictions
```

Comment:
`converting the` -> `the conversion of` maybe?
begin -> `begin` or `the beginning`,
end -> `end` or `the end`